rules/__tests__/prefer-to-contain.test.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+const RuleTester = require('eslint').RuleTester;
+const rule = require('../prefer-to-contain');
+
+const ruleTester = new RuleTester();
+
+ruleTester.run('prefer-to-contain', rule, {
+  valid: [
+    'expect(a).toContain(b);',
+    "expect(a.name).toBe('b');",
+    'expect(a).toBe(true);',
+    `expect(a).toEqual(b)`,
+    `expect(a.test(c)).toEqual(b)`,
+    `expect(a.includes(b)).toEqual(false)`,
+    `expect(a.includes(b)).toEqual()`,
+    `expect(a.includes(b)).toEqual("test")`,
+    `expect(a.includes(b)).toBe("test")`,
+    `expect(a.includes()).toEqual()`,
+    `expect(a.includes(b)).toBe(false)`,
```

Comment:
could we support this, and suggest `expect(a).not.toContain(b)`?

Reply:
Agree on supporting the `.not.toContain()` syntax. 👍

Reply:
Added `not.toContain` suggestions